docs/docs/installation-dc/quickstart.md

```diff
@@ -30,7 +30,7 @@ Installation should only take a few minutes.
 To install the newest version of Mathesar, cut-and-paste the below command into a terminal and follow the instructions:
 
 ```sh
-bash <(curl -sL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
+bash <(curl -sSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
```

Comment:
```suggestion
bash <(curl -sfSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
```
This way, curl actually fails, and exits non-zero. Otherwise, the effect is to redirect the text of the 404 page (or whatever error, if it produces text) into `bash`.

Reply:
It works!